Devscord.DiscordFramework/Services/UsersService.cs

```diff
@@ -41,6 +41,11 @@ public UserContext GetUserByMention(DiscordServerContext server, string mention)
             return user;
         }
 
+        public UserContext GetUserById(DiscordServerContext server, ulong userId)
+        {
+            return GetUsers(server).FirstOrDefault(x => x.Id == userId);
+        }
+
         private RestGuildUser GetUser(UserContext user, DiscordServerContext server)
```

Comment:
czemu GetUsers zwraca UserContext a tutaj masz RestGuildUser 

czepianie sie kodu który już był wcześniej, ale fajnie byłoby to poprawić, żeby była jedna konwencja